packages/cli/src/lib/snapshot/reportViewer/reportViewer.ts

```diff
@@ -27,7 +23,7 @@ export class ReportViewer {
   private operationsToDisplay: ReportViewerOperationName[];
 
   public constructor(
-    private readonly report: ResourceSnapshotsReportModel,
+    private reporter: SnapshotReporter,
```

Comment:
I don't think you're doing any re-assignation of this field, so I think you should keep the `readonly` ;) 

```suggestion
    private readonly reporter: SnapshotReporter,
```